index.html

```diff
@@ -0,0 +1,297 @@
+                        features from us.</p>
+                    <a class="figcaption__btn" href="#">Get Started</a>
+                </figcaption>
+            </figure>
+            <div class="statistics wrapper">
+                <figure class="statistics__details ">
+                    <svg class="details__svg first">
+                        <use xlink:href="./public/assets/img/statistic-section-icons.svg#icon-user"></use>
+                    </svg>
+                    <figcaption>
+                        <h6 class="details__number">90+</h6>
+                        <p class="details__description">Users</p>
+                    </figcaption>
+                </figure>
+                <img src="./public/assets/img/vertical-line.svg" alt="vertical line devides two sections">
+                <figure class="statistics__details">
+                    <svg class="details__svg">
+                        <use xlink:href="./public/assets/img/statistic-section-icons.svg#icon-location"></use>
+                    </svg>
+                    <figcaption>
+                        <h6 class="details__number">30+</h6>
+                        <p class="details__description">Locations</p>
+                    </figcaption>
+                </figure>
+                <img src="./public/assets/img/vertical-line.svg" alt="vertical line devides two sections">
+                <figure class="statistics__details">
+                    <svg class="details__svg">
+                        <use xlink:href="./public/assets/img/statistic-section-icons.svg#icon-server"></use>
+                    </svg>
+                    <figcaption>
+                        <h6 class="details__number">50+</h6>
+                        <p class="details__description">Servers</p>
+                    </figcaption>
+                </figure>
+            </div>
+
+            <figure class="main-section__figure  wrapper">
+                <img src="./public/assets/img/main-section-second-img.png"
+                    alt="a man with laptop shows signs of happiness">
+                <figcaption class="figure__figcation">
+                    <h2 class="figcaption__title">We Provide Many Features You Can Use</h2>
+                    <p class="figcaption__description">You can explore the features that we provide with fun and have
+                        their own functions each feature.
+                    </p>
+                    <ul class="figcaption__list">
+                        <li class="figcaption__list-element">Powerfull online protection.</li>
+                        <li class="figcaption__list-element">Internet without borders.</li>
+                        <li class="figcaption__list-element">Supercharged VPN</li>
+                        <li class="figcaption__list-element">No specific time limits.</li>
+                    </ul>
+                </figcaption>
```

Comment:
це можна як окремий блок брати
```suggestion
            <figure class="main-sectioт-figure  wrapper">
                <img src="./public/assets/img/main-section-second-img.png"
                    alt="a man with laptop shows signs of happiness">
                <figcaption class="main-sectioт-figure__figcation">
                    <h2 class="main-sectioт-figure__title">We Provide Many Features You Can Use</h2>
                    <p class="figcaption__description">You can explore the features that we provide with fun and have
                        their own functions each feature.
                    </p>
                    <ul class="main-sectioт-figure__list">
                        <li class="main-sectioт-figure__list-element">Powerfull online protection.</li>
                        <li class="main-sectioт-figure__list-element">Internet without borders.</li>
                        <li class="main-sectioт-figure__list-element">Supercharged VPN</li>
                        <li class="main-sectioт-figure__list-element">No specific time limits.</li>
                    </ul>
                </figcaption>
```